Can't send File between threads safely

```shell
error[E0277]: `*mut hdfs_sys::hdfs_2_2::hdfs_internal` cannot be sent between threads safely
  --> src/services/hdfs/backend.rs:54:12
   |
54 |         Ok(Box::new(f))
   |         -- ^^^^^^^^^^^ `*mut hdfs_sys::hdfs_2_2::hdfs_internal` cannot be sent between threads safely
   |         |
   |         required by a bound introduced by this call
   |
   = help: within `hdrs::File`, the trait `std::marker::Send` is not implemented for `*mut hdfs_sys::hdfs_2_2::hdfs_internal`
   = note: required because it appears within the type `hdrs::File`
note: required because of the requirements on the impl of `BytesRead` for `hdrs::File`
  --> src/io.rs:26:9
   |
26 | impl<T> BytesRead for T where T: AsyncRead + Unpin + Send {}
   |         ^^^^^^^^^     ^
   = note: required for the cast to the object type `dyn BytesRead`

error[E0277]: `*mut hdfs_sys::hdfs_2_2::hdfsFile_internal` cannot be sent between threads safely
  --> src/services/hdfs/backend.rs:54:12
   |
54 |         Ok(Box::new(f))
   |         -- ^^^^^^^^^^^ `*mut hdfs_sys::hdfs_2_2::hdfsFile_internal` cannot be sent between threads safely
   |         |
   |         required by a bound introduced by this call
   |
   = help: within `hdrs::File`, the trait `std::marker::Send` is not implemented for `*mut hdfs_sys::hdfs_2_2::hdfsFile_internal`
   = note: required because it appears within the type `hdrs::File`
```

Should hdrs needs to implement a thread-safe file?

--- a/src/client.rs
+++ b/src/client.rs
@@ -34,6 +34,10 @@ pub struct Client {
     fs: hdfsFS,
 }
 
+/// HDFS's client handle is thread safe.
+unsafe impl Send for Client {}
+unsafe impl Sync for Client {}
+
 impl Client {
     /// Connect to a name node with port
     ///

--- a/src/file.rs
+++ b/src/file.rs
@@ -29,6 +29,10 @@ pub struct File {
     f: hdfsFile,
 }
 
+/// HDFS's client handle is thread safe.
+unsafe impl Send for File {}
+unsafe impl Sync for File {}
+
 impl Drop for File {
     fn drop(&mut self) {
         unsafe {

